tests/NuGet.IndexingTests/TestSupport/MockTopDocs.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Lucene.Net.Search;
+
+namespace NuGet.IndexingTests.TestSupport
+{
+    public class MockTopDocs : TopDocs
+    {
+        public MockTopDocs(int totalHits, ScoreDoc[] scoreDocs, float maxScore) :
+            base(totalHits, scoreDocs, maxScore)
```

Comment:
Good question. This was originally done with the expectation that things would have to be overridden, but that doesn't seem to be the case and it can probably safely be removed.

That being said, I do like the clear denomination between the "mock" objects and the real objects, and this makes it easier to override if we need to later.

Either way, I do not feel particularly strongly about it, and will change it if we think it is the right thing to do.